localisation/00_wc_sad.csv

```diff
@@ -749,7 +749,21 @@ RULE_STATIC_PORTRAITS_ENABLED;Enabled;;Aktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_ENABLED_DESC;Static portraits for certain lore characters are �Genabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Gaktiviert�!.;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED;Disabled;;Deaktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED_DESC;Static portraits for certain lore characters are �Rdisabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Rdeaktiviert�!.;;;;;;;;;;;x
-;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+writing_whispers;Hears Whispers;;;;;;;;;;;;;x
+writing_whispers_desc;This character hears whispers of the Old Gods.;;;;;;;;;;;;;x
+whispers_tooltip_tip;Quality of the book depends on amount of your �YTrueReligion.GetPietyName�!.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10200;The whispers of your masters have been getting stronger recently. They are pouring their eldritch knowledge into your mind but it's simply too much for your mind to take and you're unable to make sense of any of it. You need to focus on one portion of them and write them down, but first you need a writing material.;;;;;;;;;;;;;x
+EVTOPTA_WCCSO.10200;That one servant girl had such a smooth skin.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10201;After spending some time, trying to filter out one of the voices and gathering all needed materials, you can now write down its whispers.;;;;;;;;;;;;;x
+EVTOPTA_WCCSO.10201;It speaks of wars from long-forgotten eons.;;;;;;;;;;;;;x
+EVTOPTB_WCCSO.10201;It speaks of temple-cities made of black obsidian.;;;;;;;;;;;;;x
+EVTOPTC_WCCSO.10201;It speaks of a creature with thousand forms.;;;;;;;;;;;;;x
+EVTOPTD_WCCSO.10201;It speaks of forgotten starry secrets.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTOPTD_WCCSO.10201;It speaks of forgotten secrets.;;;;;;;;;;;;;x
```

Reply:
I think, he used "starry" here to refer to stars, right? At least that's why I didn't mention this.

Reply:
The starry seems a little random though. If it was intended to refer exclusively to stars, it should be 
```suggestion
EVTOPTD_WCCSO.10201;It speaks of forgotten secrets of the stars.;;;;;;;;;;;;;x
```

Reply:
I was thinking of this https://en.wikipedia.org/wiki/The_Starry_Night and found it fitting :)

Reply:
That's fair. Leave it as it is then.